src/instructlab/model/evaluate.py

```diff
@@ -139,18 +139,28 @@ def validate_model(model: str, model_arg: str = "--model"):
         raise click.exceptions.Exit(1)
 
 
-def sort_score(pairing: tuple[str, float]) -> float:
+def sort_score(pairing: tuple[str, float, float, float]) -> float:
     """helper func for display_branch_eval_summary
     takes a tuple pairing and returns just the score
     """
     return pairing[1]
 
 
-def display_models(model, base_model) -> None:
+def display_models_and_scores(
+    benchmark, model, base_model, model_overall_score, base_model_overall_score
+) -> None:
     """prints the base_model and model with a header"""
-    print("## BASE MODEL")
-    print(base_model)
-    display_model(model)
+    # total score for MT-BENCH-BRANCH
+    total_score = "10.00"
```

Comment:
Debating whether this looks better with 10.00 vs 10.0 or 10

In the output it's showing up as 10.0/10.00 which looks a little off.